### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by krak3n77** [xMasterX/flipper-pager#4]
Would it be possible to add FLEX pager support?

**Comment by davidslyons** [comment]
> Would it be possible to add FLEX pager support?

I'll second this

**Comment by Geofferey** [comment]
FLEX is very active in my area as well! Unfortunately I have been unable to find any valid POCSAG freqs. I planned to put a feature request and saw this. 

I wonder if something like this would help. 

https://github.com/EliasOenal/multimon-ng/blob/master/demod_flex.c

**Comment by krak3n77** [comment]
https://github.com/detectingdragon77/flipper-pager-flex made a messy fork, replaced the pocsag stuff with flex stuff, thinking it might need a diff subghz preset than whats available.

**Comment by Geofferey** [comment]
@detectingdragon77 Great! Now I just gotta figure out how to build. Thank you for the work you're putting on this!

EDIT:

That was pretty easy, just git cloned into the firmware build and it got done. You're probably right about the preset. I can't get it to sound complete right with any of presets in RogueMaster. FM95 is best sounding.

**Comment by krak3n77** [comment]
> @detectingdragon77 Great! Now I just gotta figure out how to build. Thank you for the work you're putting on this!
> 
> EDIT:
> 
> That was pretty easy, just git cloned into the firmware build and it got done. You're probably right about the preset. I can't get it to sound complete right with any of presets in RogueMaster. FM95 is best sounding.

Yes I have found that FM95 and FM15k work the best in terms of audio, I also think that if there's no audio theres no radio signal coming in, but I could be wrong.
I am from north America, and so far I've only found flex pagers, they are really strong up in the 929mhz, as strong as FM radio stations.

**Comment by Geofferey** [comment]
I was able to record a .raw FLEX1600 periodic test signal with FM95 and play it back to multimon-ng which successfully decodes. Albeit the signal is at 929.462Mhz which is very slightly out of CC1101's range so I played back on 928Mhz which also required me to offset frequency of SDR by +000.0110Mhz ... So in theory the FM95 preset seems able to properly hear the signal well enough. Unfortunately, I have had 0% success decoding using the Flex Pager app you produced. Have you been able to decode anything as of yet.

**Comment by krak3n77** [comment]
> I was able to record a .raw FLEX1600 periodic test signal with FM95 and play it back to multimon-ng which successfully decodes. Albeit the signal is at 929.462Mhz which is very slightly out of CC1101's range so I played back on 928Mhz which also required me to offset frequency of SDR by +000.0110Mhz ... So in theory the FM95 preset seems able to properly hear the signal well enough. Unfortunately, I have had 0% success decoding using the Flex Pager app you produced. Have you been able to decode anything as of yet.

No I have not. I realized that there is a minimum signal threshold defined in the code, will try to modify it to see if theres any results.

**Comment by Geofferey** [comment]
@detectingdragon77 ahhhhh, I see! That makes sense. When I record the .raw .sub signal shows as almost nonexistent, but I can hear it loud and clear with sound on when recording and playing back. 

Here is a .raw .sub that I just recorded [flex_seq_5052.sub](https://gist.github.com/Geofferey/9e20677fb53ce295c87dcc2820830e58). 

^ Frequency in sub has been adjusted to broadcast on 928mhz on my flipper. Seems to be calibration issue at higher edge frequencies so frequency is adjusted to 927985500 to get it close to 928 as possible.

**Comment by coppertwistedpair** [comment]
does anyone still have the 

> https://github.com/detectingdragon77/flipper-pager-flex made a messy fork, replaced the pocsag stuff with flex stuff, thinking it might need a diff subghz preset than whats available.

Does anyone still have this fork? Seems to be 404'd and id like to be able to decode FLEX on my flipper

**Comment by krak3n77** [comment]
> does anyone still have the
> 
> > https://github.com/detectingdragon77/flipper-pager-flex made a messy fork, replaced the pocsag stuff with flex stuff, thinking it might need a diff subghz preset than whats available.
> 
> Does anyone still have this fork? Seems to be 404'd and id like to be able to decode FLEX on my flipper

Applologies, there was a time when another dev and me were working on this, but I gave up. Haven't heard from the other dev, so i assumed he gave up too.

**Comment by coppertwistedpair** [comment]
> > does anyone still have the
> > > https://github.com/detectingdragon77/flipper-pager-flex made a messy fork, replaced the pocsag stuff with flex stuff, thinking it might need a diff subghz preset than whats available.
> > 
> > 
> > Does anyone still have this fork? Seems to be 404'd and id like to be able to decode FLEX on my flipper
> 
> Applologies, there was a time when another dev and me were working on this, but I gave up. Haven't heard from the other dev, so i assumed he gave up too.

Damn, would hae been fun to mess with, PDW isnt exactly portable like the flipper is

**Comment by krak3n77** [comment]
> > > does anyone still have the
> > > > https://github.com/detectingdragon77/flipper-pager-flex made a messy fork, replaced the pocsag stuff with flex stuff, thinking it might need a diff subghz preset than whats available.
> > > 
> > > 
> > > Does anyone still have this fork? Seems to be 404'd and id like to be able to decode FLEX on my flipper
> > 
> > 
> > Applologies, there was a time when another dev and me were working on this, but I gave up. Haven't heard from the other dev, so i assumed he gave up too.
> 
> Damn, would hae been fun to mess with, PDW isnt exactly portable like the flipper is

yeah i know, weird that we dont have one considering pocsag is rarely used commercially in usa. (least not where im from)
